docs/DeveloperGuide.md

```diff
@@ -709,3 +721,27 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+## **Appendix G: Future Implementations**
+
+* Contacts list are only allowed to be sorted in ascending order for NAME and APPOINTMENT_DATE prefix only. We plan to allow users to sort by descending order in the future as well.
+* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative. We plan to allow tags to be added of the existing tags or remove the tags individually.
+* 
+
+## **Appendix H: Effort**
+
+{to be added}
+
+## **Appendix I: Planned Enhancements**
+
+* The current calendar window is not dynamically updated when user change client's contact information. User would have to close and reopen the calendar window to show the updated information. We plan to allow calendar window to always listen to any changes that occur to the database and automatically update the information shown in the calendar window. 
+* The application will start to experience lag after prolonged usage. This is most likely due to the application keeping the state of the database after every commands to be able revert to previous state if need be. In the future, we might plan to limit the amount of states that is kept to reduce the storage load of the application. 
```

Comment:
@glenngnng, can help to confirm/change this part out

Reply:
Hmm, actually it is possible for lag to occur even without the storing of states. States will only be stored when a redo command is called. Perhaps, it is due to the extra storing of persons whenever a command modifies the address book. For instance, doing an add command will store the person in the address book, and also in the addedPersons ArrayList in ModelManager.java. Similarly for delete, clear, edit. Furthermore, these commands are also stored in another previousUndoableCommands ArrayList. As extra memory needs to be dedicated to such storage, this can be a reason for the lag after a large number (lets say 100) commands that modify the address book.

Reply:
alright! I updated the description